ios-baseUnitTests/Extensions/StringExtensionUnitTests.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  StringExtensionUnitTests.swift
+//  ios-baseUnitTests
+//
+//  Created by German on 4/30/20.
+//  Copyright © 2020 TopTier labs. All rights reserved.
+//
+
+import XCTest
+@testable import ios_base_Debug
+
+class StringExtensionUnitTests: XCTestCase {
+  func testEmailValidation() {
+    XCTAssertFalse("username".isEmailFormatted())
+    XCTAssertFalse("username@test".isEmailFormatted())
+    XCTAssert("username@test.com".isEmailFormatted())
+    XCTAssert("username.alias+2@gmail.com".isEmailFormatted())
```

Comment:
I think that there is something wrong with the regex we have to check the email format. It's returning that strings like `some@mail.c` are valid emails, while I think that top-level domains should be at least 2 characters long.

Reply:
There is no restrictions in the standard for single character TLDs. So it would be possible to have one just that currently there is none registered. Anyways, this is not related to these changes and if we determine that we should avoid single character TLD we should open an issue for that.